ttnn/cpp/ttnn/operations/ccl/all_gather/device/all_gather_op.cpp

```diff
@@ -15,14 +15,14 @@ namespace ttnn {
 
 
 AllGatherBidirectionalMode AllGatherConfig::choose_bidirectional_mode(Tensor const& input_tensor) {
+    return AllGatherBidirectionalMode::FULL_TENSOR;
     std::size_t eth_l1_capacity = eth_l1_mem::address_map::MAX_L1_LOADING_SIZE - eth_l1_mem::address_map::ERISC_L1_UNRESERVED_BASE;
```

Comment:
do we need the call after return? :)

Reply:
The early return should be removed here and for now we should be parametrizing on if fusion is enabled. If Fusion enabled, always use FullTensor

Reply:
done